utils/getInvoices/tests/getInvoices.discount.test.ts

```diff
@@ -295,157 +301,13 @@ describe('getInvoices - DISCOUNT', () => {
-      })
-    })
-    it('should NOT load when service = null, company = null, prevPayment = { invoice: [Electricity] }', () => {
-      const service: Partial<IService> = null
-      const company: Partial<IRealestate> = null
-      const prevPayment: Partial<IPayment> = {
-        invoice: [
-          {
-            type: ServiceType.Electricity,
-            price: 1000,
-            sum: 1000,
-          },
-        ],
-      }
-
-      const invoices = getInvoices({
-        service,
-        company,
-        prevPayment,
-      })
-
-      expect(invoices).not.toContainEqual({
-        type: ServiceType.Discount,
-      })
-    })
-    it('should NOT load when service = { rentPrice: 1000 }, company = { discount: 1000 }, prevPayment = { invoice: [Discount] }', () => {
-      const service: Partial<IService> = {
-        rentPrice: 1000,
-      }
-      const company: Partial<IRealestate> = {
-        discount: 1000,
-      }
-      const prevPayment: Partial<IPayment> = {
-        invoice: [
-          {
-            type: ServiceType.Discount,
-            price: 1000,
-            sum: 1000,
-          },
-        ],
-      }
-
-      const invoices = getInvoices({
-        service,
-        company,
-        prevPayment,
-      })
-
-      expect(invoices).not.toContainEqual({
-        type: ServiceType.Discount,
-      })
-    })
-    it('should NOT load when service = { rentPrice: 1000 }, company = { discount: 1000 }, prevPayment = { invoice: [Electricity] }', () => {
-      const service: Partial<IService> = {
-        rentPrice: 1000,
-      }
-      const company: Partial<IRealestate> = {
-        discount: 1000,
-      }
-      const prevPayment: Partial<IPayment> = {
-        invoice: [
-          {
-            type: ServiceType.Electricity,
-            price: 1000,
-            sum: 1000,
-          },
-        ],
-      }
-
-      const invoices = getInvoices({
-        service,
-        company,
-        prevPayment,
-      })
-
-      expect(invoices).not.toContainEqual({
-        type: ServiceType.Discount,
-      })
-    })
-    it('should load when service = { ... }, company = { discount: 1000 }, prevPayment = { invoice: [Discount] }', () => {
-      const service: Partial<IService> = {
-        rentPrice: 1000,
-      }
-      const company: Partial<IRealestate> = {
-        discount: 1000,
-      }
-      const prevPayment: Partial<IPayment> = {
-        invoice: [
-          {
-            type: ServiceType.Discount,
-            price: 1000,
-            sum: 1000,
-          },
-        ],
-      }
-
-      const invoices = getInvoices({
-        service,
-        company,
-        prevPayment,
-      })
-
-      expect(invoices).toContainEqual({
-        type: ServiceType.Discount,
-        price: 1000,
-        sum: 1000,
-      })
+      expect(invoices).toContainEqual(
+        expect.objectContaining({
+          type: ServiceType.Discount,
+          price: 1000,
+          sum: 1000,
+        })
+      )
```

Comment:
No `expect.objectContaining()` on 'shoud load' tests. We should receive exact result